pkgs/development/python-modules/tempest/default.nix

```diff
@@ -36,9 +36,13 @@ buildPythonPackage rec {
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "sha256-aEtBAE3p+HVw/macwZtKo20mSJctrsIN7idqWe6Dvtc=";
+    hash = "sha256-aEtBAE3p+HVw/macwZtKo20mSJctrsIN7idqWe6Dvtc=";
   };
 
+  nativeBuildInputs = [
+    pbr
```

Comment:
It's still listed in `propagatedBuildInputs`, is that intended (from the commit message I'd guess no)

Reply:
No, should be removed.

Reply:
this is used at runtime https://github.com/openstack/tempest/blob/1569290be06e61d63061ae35a997aff0ebad68f1/tempest/version.py#L16-L18